entities/Rules/index.js

```diff
@@ -321,6 +291,113 @@ module.exports = {
       UI.payingRent(gameState.currentPlayer, owner, rentAmount);
     },
   ],
+  MANAGE_PROPERTIES: [
+    // TODO: potentially refactor this prompt pattern
+    ({ notify, UI }, gameState) => {
+      const action = require('../PlayerActions').prompt(
+        { notify, UI },
+        gameState,
+        require('../PropertyManagementService').getAvailableManagementActions(
+          gameState
+        )
+      );
+
+      if (action === 'CANCEL') return;
+
+      if (action) {
+        notify(action);
+      } else {
+        UI.unknownAction();
+      }
+
+      notify('MANAGE_PROPERTIES');
+    },
+  ],
+  RENOVATE: [
+    ({ notify, UI }, gameState) => {
+      const renoProps = require('../PropertyManagementService').getRenoProperties(
+        gameState
+      );
+      const propSelection = require('../PlayerActions').prompt(
+        { notify, UI },
+        gameState,
+        [...renoProps.map((p) => p.name), 'CANCEL']
+      );
+
+      if (propSelection === 'CANCEL') return;
+
+      if (propSelection) {
+        const propToReno = renoProps.find((p) => p.name === propSelection);
+        require('../PropertyManagementService').renovate(gameState, propToReno);
+      } else {
+        UI.unknownAction();
+      }
+
+      notify('RENOVATE');
+    },
+  ],
+  DEMOLISH: [
+    ({ notify, UI }, gameState) => {
+      const demoProps = require('../PropertyManagementService').getDemoProperties(
+        gameState
+      );
+      const propSelection = require('../PlayerActions').prompt(
+        { notify, UI },
+        gameState,
+        [...demoProps.map((p) => p.name), 'CANCEL']
+      );
+
+      if (propSelection === 'CANCEL') return;
+
+      if (propSelection) {
+        const propToReno = demoProps.find((p) => p.name === propSelection);
+        require('../PropertyManagementService').demolish(gameState, propToReno);
+      } else {
+        UI.unknownAction();
+      }
+
+      notify('DEMOLISH');
+    },
+  ],
+  MORTGAGE: [
+    ({ notify, UI }, gameState) => {
+      const player = gameState.currentPlayer;
+
+      const INTEREST_RATE_MULTIPLIER = 1.1;
+      // can only mortgage or unmortgage properties with 0 buildings
+      const mortgageAbleProps = require('../PropertyManagementService')
+        .getProperties(gameState)
+        .filter((p) => p.ownedBy === player.id && p.buildings === 0);
+      const propSelection = require('../PlayerActions').prompt(
+        { notify, UI },
+        gameState,
+        [...mortgageAbleProps.map((p) => p.name), 'CANCEL']
+      );
+
+      if (propSelection === 'CANCEL') return;
+
+      if (propSelection) {
+        const mortgageProp = mortgageAbleProps.find(
+          (p) => p.name === propSelection
+        );
+        if (
+          mortgageProp.mortgaged &&
+          player.cash < (mortgageProp.price / 2) * INTEREST_RATE_MULTIPLIER
```

Comment:
Wasn't the mortgaged price already `/ 2`'d ?

Reply:
Mortgage price is not /2, this is a pre-check to make sure they have the money to unmortgage it.

We never change the price, only the asset value